tuning_configs/lstm_tuning_config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "population_size": 6,
+    "n_generations": 4,
+    "attributes": {
+        "hidden_state_size": {
+            "type": "int",
+            "min": 16,
+            "max": 512,

```

Comment:
The ranges in this file are way too big for a unit test. We should be using something like min: 1, max:2.